wd_demo.au3

```diff
@@ -51,34 +51,50 @@ Exit
 
 Func _WD_Demo()
 	Local $nMsg
-	Local $iSpacing = 50
+	Local $iPos
+	Local $iSpacing = 25
 	Local $iCount = UBound($aDemoSuite)
 	Local $aCheckboxes[$iCount]
 
-	Local $hGUI = GUICreate("Webdriver Demo", 200, 150 + (20 * $iCount), 100, 200, BitXOR($GUI_SS_DEFAULT_GUI, $WS_MINIMIZEBOX))
+	Local $hGUI = GUICreate("Webdriver Demo", 200, 100, 100, 200, BitXOR($GUI_SS_DEFAULT_GUI, $WS_MINIMIZEBOX))
```

Comment:
This was coded so that the GUI would dynamically expand as additional demo routines were added. Why would you want to remove this "feature"?

Reply:
focus on:
```AutoIt
	; Resize window
	WinMove($hGUI, "", 100, 200, 200, $iPos + 3 * $iSpacing)
```

I think this is a better way to achieve this what you are talking about.